mlflow/utils/autologging_utils/versioning.py

```diff
@@ -0,0 +1,50 @@
+import importlib
```

Comment:
Moved supported version checking / validation logic from `mlflow/utils/autologging_utils/__init__.py` and updated some docstrings & variable / method names.